index.html

```diff
@@ -15,7 +15,7 @@
 
     <header class="bg-violet-500">
       <div class="container mx-auto px-5 py-8 flex flex-row justify-between">
-        <img class="h-14" src="./images/osslogo-text-white.png" alt="">
+        <a href="https://ossph.org/" target="_blank"><img class="h-14" src="./images/osslogo-text-white.png" alt=""></a>
```

Comment:
Hi, all good. I just have one last request. Please add "OSSPH Logo" as `alt` in the image. Thanks.

Reply:
Done. Kindly please review it, Thank you.